src/bp/core/repositories/users.ts

```diff
@@ -167,6 +182,8 @@ export class KnexUserRepository implements UserRepository {
       .knex(this.tableName)
       .update({ attributes: this.database.knex.json.set({ ...originalAttributes, ...attributes }) })
       .where({ channel, user_id })
+
+    this.userCache.del(this.getCacheKey(channel, user_id))
```

Comment:
This is not distributed and updating attributes will only invalidate attributes on the active node, while other nodes will give an incorrect value for attributes